modules/get_me_some_ice_cream/utils.py

```diff
@@ -0,0 +1,22 @@
+def insert_user_name(text, username):
+    return text.replace('$$', username)
+
+
+def extract_scoops(prop_ner):
```

Comment:
This can be moved to detection.py, then this file can be removed. Not a fan of generic utilility files, since they have a tendency to absorb everything and their grandmas.